components/docs-chef-io/content/automate/reusable/md/restore_troubleshooting.md

```diff
@@ -19,3 +19,173 @@ Try these steps if Chef Automate returns an error while restoring data.
     ```
 
 Now check the status of the Automate node and then try running the restore command from the bastion host.
+
+1. How to change the `base_path` or `path`. Below steps for the File System backup
+   - While at the time of deployment `backup_mount` default value will be `/mnt/automate_backups`
+   - In case, if you modify the `backup_mount` in `config.toml` before deploment, then deployment process will do the configuration with updated value
+   - In case, you changed the `backup_mount` value post deployment, then we need to patch the configuration manually to all the frontend and backend nodes, for example if you change the `backup_mount` to `/bkp/backps`
+   - Update the FE nodes with below template, use command `chef-automate config patch fe.toml --fe`
+
+      ```sh
+         [global.v1.backups]
+            [global.v1.backups.filesystem]
+               path = "/bkp/backps"
+         [global.v1.external.opensearch.backup]
+            [global.v1.external.opensearch.backup.fs]
+               path = "/bkp/backps"
+      ```
+
+   - Update the opensearch node with below template, use command `chef-automate config patch os.toml --os`
+
+      ```sh
+      [path]
+         repo = "/bkp/backps"
+      ```
+
+      - Run the curl request to one of the automate frontend node
+
+        ```sh
+        curl localhost:10144/_snapshot?pretty
+        ```
+
+         - If the response is empty `{}`, then we are good
+
+         - If the response has json output, then it should have correct value for the `backup_mount`, refer the `location` value in the response. It should start with the `/bkp/backps`
+
+        ```sh
+        {
+         "chef-automate-es6-event-feed-service" : {
+         "type" : "fs",
+         "settings" : {
+         "location" : "/mnt/automate_backups/opensearch/automate-elasticsearch-data/chef-automate-es6-event-feed-service"
+               }
+            },
+         "chef-automate-es6-compliance-service" : {
+         "type" : "fs",
+         "settings" : {
+         "location" : "/mnt/automate_backups/opensearch/automate-elasticsearch-data/chef-automate-es6-compliance-service"
+               }
+            },
+         "chef-automate-es6-ingest-service" : {
+             "type" : "fs",
+         "settings" : {
+         "location" : "/mnt/automate_backups/opensearch/automate-elasticsearch-data/chef-automate-es6-ingest-service"
+               }
+            },
+         "chef-automate-es6-automate-cs-oc-erchef" : {
+         "type" : "fs",
+         "settings" : {
+         "location" : "/mnt/automate_backups/opensearch/automate-elasticsearch-data/chef-automate-es6-automate-cs-oc-erchef"
+               }
+            }
+         }
+        ```
+
+         - If the pre string in the `location` is not match with `backup_mount`, then we need to to delete the existing snapshots. use below script to delete the snapshot from the one of the automate frontend node.
```

Comment:
- If the pre string in the `location` is not matched with `backup_mount`, then we need to delete the existing snapshots. You can use the below script to delete the snapshot from one of the automate frontend nodes.